sns/cm/dtrebalance/cm.c

```diff
@@ -231,7 +245,7 @@ M0_INTERNAL const struct m0_cm_ops sns_dtrebalance_ops = {
 	.cmo_ag_alloc            = m0_sns_cm_dtrebalance_ag_alloc,
 	.cmo_cp_alloc            = dtrebalance_cm_cp_alloc,
 	.cmo_data_next           = m0_sns_cm_iter_next,
-	.cmo_ag_next             = m0_sns_cm_ag_next,
+	.cmo_ag_next             = dtrebalance_cm_ag_next,
```

Comment:
To keep function names in sync, how about renaming this function to m0_sns_dtrebalance_cm_ag_next ?

Reply:
function names should begin with m0_* only if they are exposed outside. This is how they are named in existing repair and rebalance code.